configure.ac

```diff
@@ -1,4 +1,4 @@
-AC_INIT([libzbxpgsql], [1.1.0], [ryan@cavaliercoder.com])
+AC_INIT([libzbxpgsql], [1.1.1], [ryan@cavaliercoder.com])
```

Comment:
1.1.0 hasn't been released yet, so this can stay as 1.1.0. I intend to include your changes in 1.1.0.